pkgs/development/libraries/stargate-libcds/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "stargate-libcds";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "stargateaudio";
+    repo = "libcds";
+    rev = version;
+    sha256 = "sha256-THThEzS8gGdwn3h0EBttaX5ljZH9Ma2Rcg143+GIdU8=";
+  };
+
+  # Remove unecessary tests (valgrind, coverage)
+  patches = [ ./Makefile.patch ];
+
+  doCheck = true;
+
+  installPhase = ''
+    install -D libcds.so -t $out/usr/lib/
+  '';
```

maybe this should be installed to `$out/lib` instead?
```suggestion
  installPhase = ''
    runHook preInstall
    install -D libcds.so -t $out/usr/lib/
    runHook postInstall
  '';
```

---

i'm not sure. afaik /usr/lib is for user programs under /usr/bin (like stargate?), and /lib is for essential libraries for the system. more info at https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/lib.html and https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/usr.html

---

Nix doesn't make use FHS.

All dynamic libraries should be located under `${!outputLib}/lib`, in this case, it's `$out/lib`

---

@jonringer copy that. done!